sympy/integrals/integrals.py

```diff
@@ -342,7 +347,7 @@ def _calc_limit(a, b):
                 if len(xab) == 3:
                     a, b = xab[1:]
                     a, b = _calc_limit(a, b), _calc_limit(b, a)
-                    if a - b > 0:
+                    if (a - b > 0) in (True, true):
```

Comment:
How about `if fuzzy_bool(a > b)`?

Reply:
Might work! Never used fuzzy_bool... I'll try to look into it.

Reply:
That's because x isn't known to be real:
```julia
In [6]: x = Symbol('x', real=True)

In [7]: x+1 > x+2
Out[7]: False
```
See #16915

Reply:
I kept the current one as the integration variables should be real, at least in this case, although not explicitly stated. Once #16014 is solved (or something similar), it should be possible to change to `a > b` without any side effects (admittedly keeping it only solves a somewhat singular case, but if nothing else there is an existing test with exactly that).